### PR TITLE
Chore/linux e2e

### PR DESCRIPTION
**Pull request by tejohnso**
@ahmedalsudani @fjvallarino please review

### CODE REVIEW

**Comment by tejohnso** [comment]
@ahmedalsudani @fjvallarino squashed to one commit - please review

**Comment by fjvallarino** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -13,7 +13,9 @@ npm start
 
 *md5* contains the expected md5 result of the display's presentation screenshot:  eg `"e1 db 21 c2 16 ea bf 6b 11 84 bd 8d 0a 3d d9 5d"`
 
-an md5 can be generated via `node screenshot-test.js`
+On Windows an md5 can be generated via `node screenshot-test.js`
+
+On Linux `scrot -d 5` and `md5sum` can be used.
```
Comment (ahmedalsudani) [inline README.md]: @tejohnso Just curious, why can't we use `node screenshot-test.js` on Linux?

File: downloader.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 const path = require("path"),
 os = process.platform === "linux" ? "lnx" : "win",
-downloadedInstallerFileName = "installer" + (os === "lnx" ? ".sh" : ".exe"),
+fileSuffix = (os === "lnx" ? "sh" : "exe"),
+downloadedInstallerFileName = "installer." + fileSuffix,
 arch = process.arch === "x64" ? "64" : "32",
-baseUrl = "http://install-versions.risevision.com/VERSION/installer-OS-ARCH.exe",
+baseUrl = "http://install-versions.risevision.com/VERSION/installer-OS-ARCH.SUFFIX",
```
Comment (ahmedalsudani) [inline downloader.js]: @tejohnso can we not use ES6 string interpolation here?

File: README.md (hunk 1 above)
Comment (tejohnso) [inline README.md]: @ahmedalsudani On linux it's easy to create the screenshot without the script so I didn't bother cross-platformifying it.

File: downloader.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ module.exports = {
     const file = fs.createWriteStream(downloadedInstallerFileName);
 
     return new Promise((res, rej)=>{
-      sendRequest(baseUrl.replace("VERSION", version).replace("OS", os).replace("ARCH", arch).replace("SUFFIX", fileSuffix));
+      sendRequest(`${baseUrl}${version}/installer-${os}-${arch}.${fileSuffix}`);
```
Comment (tejohnso) [inline downloader.js]: @ahmedalsudani updated

**Comment by ahmedalsudani** [comment]
:+1:

**Pull request merged**